src/org/usfirst/frc4904/robot/RobotMap.java

```diff
@@ -57,55 +69,79 @@
 		
 		public static class PCM {}
 		
-		public static class Pneumatics {}
+		public static class Pneumatics {
+			public static int solenoidUp = 0;
+			public static int solenoidDown = 1;
+		}
+	}
+	
+	public static class Constant {
```

Comment:
Suggested: Remove the Constants class completely, including all chassis variables. We do not use them yet.